prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -1106,5 +1106,51 @@ public async Task<ActionResult<EnrolleeNoteViewModel>> DeleteSiteNotifications(i
 
             return Ok();
         }
+
+        // PUT: api/sites/5/flag
+        /// <summary>
+        /// Sets a site's flag, which serves as a reminder
+        /// for an adjudicator to come back to this site
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpPut("{siteId}/flag", Name = nameof(FlagSite))]
+        [Authorize(Roles = Roles.ViewSite)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status200OK)]
+        public async Task<ActionResult> FlagSite(int siteId)
+        {
+            return await FlagSiteInternal(siteId, true);
+        }
+
+        // DELETE: api/sites/5/flag
+        /// <summary>
+        /// Removes a site's flag, indicating the adjudicator
+        /// is done working on it/has no need for the site flagged
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpDelete("{siteId}/flag", Name = nameof(RemoveFlagSite))]
```

Comment:
feels like should use http verb PUT instead of DELETE, since the site resource is not deleted but modified.